Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -0,0 +1,228 @@
+             * Listened in {eZ.Plugin.DashboardBlockAllContentService}
+             *
+             * @event getAllContent
+             */
+            this.fire('getAllContent');
+        },
+
+        /**
+         * Renders dashboard block content
+         *
+         * @method _renderContent
+         * @protected
+         * @param event {Object} event facade
+         * @param event.newVal {Y.LazyModelList} list of content models
+         * @return {eZ.DashboardBlockAllContentView} the view itself
+         */
+        _renderContent: function () {
+            var contentList = this.get('contentList'),
+                fragment = Y.one(document.createDocumentFragment()),
+                rows = [],
+                RowView = this.get('rowView'),
+                renderRow = Y.bind(function (data) {
+                    var row = new RowView({
+                        content: data.content,
+                        contentType: data.contentType
+                    });
+
+                    row.addTarget(this);
+                    rows.push(row);
+                    fragment.append(row.render().get('container'));
+                }, this);
+
+            if (!contentList) {
+                return;
+            }
+
+            this._destroyRows(true);
+            contentList.forEach(renderRow);
+
+            this.get('container').one(SELECTOR_CONTENT).setHTML(fragment);
+            this.set('rows', rows);
+
+            return this;
+        },
+
+        /**
+         * Destroys existing rows
+         *
+         * @method _destroyRows
+         * @protected
+         * @param reset {Object} should the rows info be reset?
+         */
+        _destroyRows: function (reset) {
+            var rows = this.get('rows');
+
+            rows.forEach(Y.bind(function (row) {
+                row.removeTarget(this);
+                row.destroy({remove: true});
+            }, this));
+
+            if (reset) {
+                this.set('rows', []);
+            }
+        },
+
+        destructor: function () {
+            this._destroyRows();
+        }
+    }, {
+        ATTRS: {
+            /**
+             * The title
+             *
+             * @attribute title
+             * @type String
+             * @default 'All Content'
+             */
+            title: {
+                value: 'All Content'
+            },
+
+            /**
+             * The dashboard block content list.
+             * Each entry contains content model and content type model.
+             *
+             * @attribute contentList
+             * @type Array
+             */
+            contentList: {},
+
+            /**
+             * The block identifier
+             *
+             * @attribute identifier
+             * @type String
+             * @default 'dashboard-all-content-block'
+             */
+            identifier: {
+                value: 'dashboard-all-content-block'
```

Comment:
I would not redefine the attribute here but rather set its value in `initializer` (with `_set` since it is supposed to be `readOnly`). Also `all-content` (or `everyone-s-content` to fit with the specs) would be enough.